scenario-runner/README.md

```diff
@@ -0,0 +1,5 @@
+# CARLA ScenarioRunner scenarios
+
+This directory contains custom ScenarioRunner scenario configurations to
+facilitate integration testing. This still a work in progress, so the scenarios
+serve more as example references.
```

Comment:
Can we include a link to CARLA Scenario Runner documentation (https://carla-scenariorunner.readthedocs.io/en/latest/)